src/adaptation/icap/ModXact.cc

```diff
@@ -248,12 +248,16 @@ void Adaptation::Icap::ModXact::writeMore()
 
     case State::writingInit:    // waiting for service OPTIONS
         Must(state.serviceWaiting);
+        return;
 
     case State::writingConnect: // waiting for the connection to establish
+        return;
 
     case State::writingHeaders: // waiting for the headers to be written
+        return;
 
     case State::writingPaused:  // waiting for the ICAP server response
+        return;
```

Comment:
Unresolved change request from Alex June 2020:
> 
> If possible, please avoid these code-duplicating changes.

State::writingInit case can use [[fallthrough]] attribute, and others can fall down to State::writingReallyDone return statement.

Reply:
I found out that in case there are multiple identical case statements, [[fallthrought]] is not even needed.
Removing it all